src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -208,5 +212,32 @@ public void WriteException(string errorMessage)
                 Console.WriteLine(log);
             }
         }
+
+        public static void Summarize(AnalysisSummary summary)
```

Comment:
>AnalysisSummary [](http://example.com/codeflow?start=37&length=15)

override the toString to facilitate the line 237

Reply:
done